src/sentry/models/groupredirect.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+from sentry.db.models import BoundedBigIntegerField, Model, FlexibleForeignKey
+
+
+class GroupRedirect(Model):
+    """
+    Maintains a reference from a group that has been merged (and subsequently
+    deleted) to the group that superceded it.
+    """
+    __core__ = False
+
+    group = FlexibleForeignKey('sentry.Group')
```

Comment:
Should probably not use a foreign key here, and just use `group_id`. Not sure we could get the lock required for the constraint sensibly.

/cc @dcramer